### DISCUSSION TITLE
Music tag not getting updated with index

### DISCUSSION THREAD

**Discussion by NikunjK8** [namidaco/namida#166] [Q&A]
So a song was already registered in Namida. Then i changed the artist from another app and tried to refresh it but Namida still shows the old tags and song doesn't appear in respective artist tab. App only refreshes the song itself not metadata. Is there any way to refresh the tags as well?

**Comment by MSOB7YY** [comment]
![Screenshot_20240411-211029_Namida](https://github.com/namidaco/namida/assets/85245079/17a1d543-72da-4cfd-9539-d4600ea03a88)
the option literally says **newly added** or **deleted** music files, in ur case you will have to select the track/tracks > advanced > re-index

**Reply by NikunjK8** [reply]
I was using swipe to refresh so didn't read this. Thanks
